content/altinn-studio/reference/ux/texts/_index.nb.md

```diff
@@ -201,7 +205,22 @@ Det er per nå mulig å hente verdier fra 3 ulike datakilder.
    1. `instanceOwnerPartyId` inneholder avgiver sin party id.
    2. `instanceId` inneholder id'en til den aktive instansen.
    3. `appId` inneholder id'en til appen instansen er knyttet til.
-    
+4. Egendefinerte tekst-parametere\
+   Ved å angi `customTextParameters` som datakilde kan man hente ut egendefinerte verdier der det er mulig å sette.
+   Per nå kan `CustomTextParameters` settes i `ValidationIssue` og `InstantiationValidationResult`. Eksempel:
+   ```cs
+    new ValidationIssue()
+    {
+        CustomTextKey = "too_long",
```

Comment:
Should this be `good.text.id` instead, to follow the example above? Or maybe this should have a more detailed example in the validation docs as well, and we could link to that here?

Reply:
The validation docs seem quite outdated, so that is another can of worms. But yeah